format_json.c

```diff
@@ -72,11 +72,11 @@ static void json_row(FILE* f, void* state, col_info_t* cols, int count) {
         if(length == SQL_NULL_DATA) {
             buffer = "null";
             length = 4;
-        } 
+        }
         else if (length == SQL_NTS) {
             length = strlen(buffer);
         }
-        
+
```

Comment:
This code can be removed here and in all the formats, since it's handled in db2util_query now.

Reply:
Right, I will clean this up

Reply:
@worksofliam can we get worksofliam/ibmi-db2util#2 merged? Then I think we are good to squash this.

Reply:
You got it @abmusse !